src/geonovum/style.js

```diff
@@ -49,7 +49,10 @@ function createStyle(css_name) {
 }
 
 function createResourceHints() {
-  const resourceHints = [
+  /**
+   * @type ResourceHintOption[]
+   * */
```

Comment:
Nit: I like these in one-line when we can :smile: 
```suggestion
  /** @type ResourceHintOption[] */
```